### PR TITLE
[OPIK-345] [UX improvements] Split feedback scores into individual columns

### PR DESCRIPTION
**Pull request by andriidudar**
## Details
image
image
image
image

## Issues

Resolves #

### CODE REVIEW

File: apps/opik-frontend/src/components/pages/PromptPage/ExperimentsTab/ExperimentsTab.tsx [hunk 1]
```diff
@@ -103,12 +132,69 @@ const ExperimentsTab: React.FC<ExperimentsTabProps> = ({ promptId }) => {
...
+
   const [columnsWidth, setColumnsWidth] = useLocalStorageState<
     Record<string, number>
   >(COLUMNS_WIDTH_KEY, {
     defaultValue: {},
   });
 
+  useEffect(() => {
+    setPresentedDynamicColumns((cols) => {
+      const dynamicColumnsIds = dynamicColumns.map((col) => col.id);
```
Comment (aadereiko) [inline apps/opik-frontend/src/components/pages/PromptPage/ExperimentsTab/ExperimentsTab.tsx:170]: maybe, you could put this logic into some function / hook. the logic here is getting harder to follow and it starts being repeated in a few files

File: apps/opik-frontend/src/components/pages/TracesPage/TracesActionsPanel.tsx [hunk 2]
```diff
@@ -53,7 +54,15 @@ const TracesActionsPanel: React.FunctionComponent<TracesActionsPanelProps> = ({
         // we need split by dot to parse usage into correct structure
         const keys = column.split(".");
         const key = last(keys) as string;
-        acc[key] = get(row, keys, "");
+        const keyPrefix = first(keys) as string;
+
+        if (keyPrefix === "feedback_scores") {
+          acc[key] = (row.feedback_scores?.find((f) => f.name === key) ?? {
```
Comment (aadereiko) [inline apps/opik-frontend/src/components/pages/TracesPage/TracesActionsPanel.tsx:60]: `(row.feedback_scores?.find((f) => f.name === key) ?? "-"` 
looks simpler, or as you had below 
`get(row.feedback_scores?.find((f) => f.name === key), "value", "-")`

**Review by aadereiko — APPROVED** [review]
Great job! Some really small code improvements - no need to tackle them now

File: apps/opik-frontend/src/components/pages/PromptPage/ExperimentsTab/ExperimentsTab.tsx (hunk 1 above)
Comment (andriidudar) [reply apps/opik-frontend/src/components/pages/PromptPage/ExperimentsTab/ExperimentsTab.tsx:170]: Good cache. I also noticed this. I will move this logic to a generic location when adding more dynamic columns across the application.

File: apps/opik-frontend/src/components/pages/TracesPage/TracesActionsPanel.tsx (hunk 2 above)
Comment (andriidudar) [reply apps/opik-frontend/src/components/pages/TracesPage/TracesActionsPanel.tsx:60]: That is a good suggestion, I will improve it with one of my next PRs.

**Pull request merged**